### ISSUE TITLE
Couldn't execute in moveit and gazebo simultaneously

### ISSUE THREAD

**Issue by kottesaikumar** [qboticslabs/mastering_ros#17]
[ERROR] [1488953201.954575053, 64.624000000]: Unable to identify any set of controllers that can actuate the specified joints: [ elbow_pitch_joint elbow_roll_joint gripper_roll_joint shoulder_pan_joint shoulder_pitch_joint wrist_pitch_joint wrist_roll_joint ]
[ERROR] [1488953201.954612200, 64.624000000]: Known controllers and their joints:

[ERROR] [1488953201.954637760, 64.624000000]: Apparently trajectory initialization failed

@qboticslabs 
I removed the first joint_state_controller from the seven_dof_arm_gazebo_control.launch
![screenshot from 2017-03-08 11-40-48](https://cloud.githubusercontent.com/assets/26217360/23692376/1db877e0-03f4-11e7-973c-8c99ca1bf030.png)

I tried installing all the controllers but it doesn't solve the problem

**Comment by qboticslabs** [comment]
@kottesaikumar  Are you getting this error everytime when you press the Plan button in Rviz? Any movement for Robot in Rviz ?

**Comment by kottesaikumar** [comment]
@qboticslabs  Yes. I installed all the controllers that were mentioned [here](https://github.com/qboticslabs/mastering_ros/issues/7) . But it still says couldn't find position_controllers

**Comment by aydal** [comment]
Hi,
Were you able to resolve the issue? I am having the same trouble.

**Comment by youliangtan** [comment]
According to the error printout on gazebo, the issue seems to be the dependencies on 'arm_controller'. I tried to install `sudo apt-get ros-melodic-joint-trajectory-controller` (change melodic if u r using kinetic), it works fine now.

**Comment by mn-banjar** [comment]
> [ERROR] [1488953201.954575053, 64.624000000]: Unable to identify any set of controllers that can actuate the specified joints: [ elbow_pitch_joint elbow_roll_joint gripper_roll_joint shoulder_pan_joint shoulder_pitch_joint wrist_pitch_joint wrist_roll_joint ]
> [ERROR] [1488953201.954612200, 64.624000000]: Known controllers and their joints:
> 
> [ERROR] [1488953201.954637760, 64.624000000]: Apparently trajectory initialization failed
> 
> @qboticslabs
> I removed the first joint_state_controller from the seven_dof_arm_gazebo_control.launch
> ![screenshot from 2017-03-08 11-40-48](https://cloud.githubusercontent.com/assets/26217360/23692376/1db877e0-03f4-11e7-973c-8c99ca1bf030.png)
> 
> I tried installing all the controllers but it doesn't solve the problem

I faced the same issue when I was trying to use an example package that uses Moveit.

In my case, I forget to install Moveit package on the ROS. so I solved the issue by running:

`sudo apt-get install ros-kinetic-moveit`
